packages/fxa-settings/src/lib/firefox.ts

```diff
@@ -20,6 +20,15 @@ export interface FirefoxMessage {
   };
 }
 
+interface ProfileUid {
+  uid: hexstring;
+}
+
+interface ProfileMetricsEnabled {
+  metricsEnabled: boolean;
+}
```

Comment:
hmm interesting that you use `interface`s here and then `type` for the sum type below.  I'd like learn more about the reason for that.

Reply:
"the doc" as in the official TS doc?